src/types/price-book-prices.d.ts

```diff
@@ -8,12 +8,8 @@ export interface PriceBookPriceBase {
   attributes: {
     currencies: {
       [key: string]: {
-        code: string
-        exchange_rate: number
-        format: string
-        decimal_point: string
-        thousand_separator: string
-        decimal_places: number
+        includes_tax: boolean
+        amount: number
```

Comment:
Please, revert those changes. The API recently was changed and the removed fields are required to create a new price